rpcs3/Emu/Cell/PPUTranslator.cpp

```diff
@@ -2429,6 +2429,11 @@ void PPUTranslator::MFOCRF(ppu_opcode_t op)
 
 void PPUTranslator::LWARX(ppu_opcode_t op)
 {
+	if (g_cfg.core.ppu_128_reservations && g_cfg.core.ppu_128_reservations_inaccurate_loop_max)
```

Comment:
Ok, what's it and why it's necessary? I thought it could be implemented without touching PPU LLVM.

Reply:
Oh, sure it can be implemented without but it's an "optimization". I should add a comment for it.
Basically it's used in ppu_load_acquire_reservation in reservation store instruction searching from current CIA of load reservation instruction.

Reply:
The shorter the atomic loop the less chance it contains a direct load instruction from reserved cache line memory so accurate reservations may not always be needed in this case.